src/assets/css/style.scss

```diff
@@ -3114,9 +3113,9 @@ Dark Home CSS
         }
     }
 
-    &.partner-area {
-        border-bottom-color: $sub-black-color;
-    }
+    // &.partner-area {
+        //Styles for partner area  
+    // }
 

```

Comment:
You can leave `&.partner-area` uncomment out and leave your inner comment if you want .... only saying this since there is no need to comment it out.

So it can be:
```
&.partner-area {
   // Styles for partner area
}
```